packages/firestore/src/platform/browser/webchannel_connection.ts

```diff
@@ -348,6 +356,16 @@ export class WebChannelConnection extends RestConnection {
       }
     );
 
+    unguardedEventListen<StatEvent>(requestStats, Event.STAT_EVENT, event => {
+      if (!event) return;
```

Comment:
Is this line necessary? If so, under what conditions would it happen. Might be worth adding a comment.

Reply:
It seems like `unguardedEventListen` can be modified safely to assume that there is always an event. Fixed.